logic/node_label.ts

```diff
@@ -265,6 +274,21 @@ export class NodeLabelHelper {
 					counts.sizeInBytes,
 					abbreviateDescriptions
 				);
+			case CountType.CustomKey:
+				if (!customKey) {
+					return null;
+				}
+				const value = counts?.frontmatter?.[customKey];
+				if (!value) {
+					return null;
+				}
+				const isValidNumber = !isNaN(Number(value))
+				const date = moment(value);
+				const text = !isValidNumber && date.isValid() ? date.format('YYYY/MM/DD') : value;
```

Comment:
Attempting to parse the value as a date seems like it would have a lot of false positives. For example, `Number(undefined)` returns NaN but `moment(undefined)` returns a valid date. If users want dates to appear, I'd prefer they format them ahead of time, by some other process. This plugin should display whatever's contained in the frontmatter key, verbatim.